### PR TITLE
Update baton_roots.html

### PR DESCRIPTION
**Pull request by DanWolfstone**
create Intro section for the web page

### CODE REVIEW

File: pages/programs/baton_roots.html [hunk 1]
```diff
@@ -1 +1,38 @@
 
+<body>
```
Comment (QuintonJason) [inline pages/programs/baton_roots.html]: Missing `<html>` tag

File: pages/programs/baton_roots.html [hunk 2]
```diff
@@ -1 +1,38 @@
...
+	<div id="header">
+		<div class="grid-st">
+			<img src="http://www.thewallsproject.org/img/shared/walls_logo.svg" width="45%" />
```
Comment (QuintonJason) [inline pages/programs/baton_roots.html]: We're using inline styles. Can the `width: 45%` be placed in the css file?

File: pages/programs/baton_roots.html [hunk 3]
```diff
@@ -1 +1,38 @@
...
+		</div>
+		<div class="grid-end">
+			<div id="nav">
+				<div class="nav-1"><a href="#">Progress<span class="u"></span></a></div>
```
Comment (QuintonJason) [inline pages/programs/baton_roots.html]: Href's not pointing to page sections. We'll add this after all sections are completed.

File: pages/programs/baton_roots.html [hunk 4]
```diff
@@ -1 +1,54 @@
+<!DOCTYPE html>
```
Comment (NekoAmbrosius) [inline pages/programs/baton_roots.html]: there are only small fixes that need to be done, nothing major

**Pull request closed without merging**